docs/tutorial_hist.rst

```diff
@@ -458,6 +458,29 @@ can find this class in the ``xonsh.teepty`` module. Like with lazy JSON, this is
 from other parts of xonsh and can be used on its own.  If you find this useful in other areas,
 please let us know!
 
+
+Sqlite backed History
+=====================
+
+Xonsh has a second built-in history backend powered by sqlite (other than
+the JSON version mentioned all above in this tutorial). It shares the same
+functionalities as the JSON version in the most ways, except it currently
```

Comment:
I would change this to: "It shares the same functionality as the JSON version in most ways, except..."